cdap-docs/developers-manual/source/building-blocks/plugins.rst

```diff
@@ -665,19 +687,20 @@ we need to expose the ``com.example.tokenizer`` package in our pom.xml:
 .. highlight:: console
 
 When deploying this artifact, we tell CDAP that the artifact extends the ``wordcount`` artifact, versions
-``1.1.0`` inclusive to ``2.0.0`` exclusive::
+``1.1.0`` inclusive to ``2.0.0`` exclusive:
+
+.. tabbed-parsed-literal::
 
- $ curl -w'\n' localhost:10000/v3/namespaces/default/artifacts/tokenizers --data-binary @tokenizers-1.0.0.jar \
-    -H 'Artifact-Extends:wordcount[1.1.0,2.0.0)'
+ $ curl -w"\n" "localhost:10000/v3/namespaces/default/artifacts/tokenizers" --data-binary @tokenizers-1.0.0.jar \
+ -H "Artifact-Extends:wordcount[1.1.0,2.0.0)"
 
 This will make the plugins available to those versions of the ``wordcount`` artifact. We can now create
-applications that use the tokenizer we want::
+applications that use the tokenizer we want:
+
+.. tabbed-parsed-literal::
 
-  $ curl -w'\n' -X PUT localhost:10000/v3/namespaces/default/apps/phrasecount -H 'Content-Type: application/json' \
-    -d '{
-      "artifact": { "name": "wordcount", "version": "1.1.0", "scope": "user" },
-      "config": { "tokenizer": "phrase" }
-    }'
+  $ curl -w"\n" -X PUT localhost:10000/v3/namespaces/default/apps/phrasecount -H "Content-Type: application/json" \
+  -d "{ 'artifact': { 'name': 'wordcount', 'version': '1.1.0', 'scope': 'user' }, 'config': { 'tokenizer': 'phrase' } }"
```

Comment:
I have had to do this in the examples (swap quotes) otherwise they don't work under Windows.